Exploration Queries/InputEntity_Account/UserAccount_SubsOwnedViaAadGroups.yaml

```diff
@@ -0,0 +1,35 @@
+Id: 3ffccd66-95b7-4437-9487-0297b40203e8
+DisplayName: "Subscriptions transitively owned by the user via AAD Group memberships"
+Description: "Returns all subscriptions owned transitively by the user via Azure Active Directory Group memberships"
+InputEntityType: Account 
+InputFields:
+  - AadUserId 
+OutputEntityTypes:
+  - AzureResource  
+QueryPeriodBefore: 7d 
+QueryPeriodAfter: 0
+DataSources:
+  - UserAccessAnalytics
+Tactics:
+  - LateralMovement 
+query: |  
+    let GetUserOwnedSubsViaGroups = ( v_Account_AadUserId:string) { 
+        let UserGroupAccess = UserAccessAnalytics
+            | where SourceEntityId =~ v_Account_AadUserId           
+            | where TargetEntityType == "Group"
+            | project TimeGenerated, GroupId = TargetEntityId, GroupName = TargetEntityName
+            | summarize arg_max(TimeGenerated, *) by GroupId, GroupName;
+        let GroupSubsAccess =   UserAccessAnalytics
+            | where TargetEntityType == "AzureSubscription"
+            | where AccessLevel == "Owner"
+            | project TimeGenerated, GroupId = SourceEntityId,  GroupName = SourceEntityName, AzureResource_Aux_SubsName = TargetEntityName, AzureResource_ResourceId = strcat("/subscriptions/", TargetEntityId), AzureResource_SubscriptionId = TargetEntityId
+            | summarize arg_max(TimeGenerated, *) by GroupId, GroupName, AzureResource_Aux_SubsName, AzureResource_ResourceId, AzureResource_SubscriptionId;
+        UserGroupAccess
+            | join kind = inner
+            GroupSubsAccess
+            on GroupId  
+            | project AzureResource_ResourceId, AzureResource_SubscriptionId, AzureResource_Aux_SubsName, GroupName, GroupId
+            | summarize AzureResource_Aux_AADGroupIds = make_set(GroupId), AzureResource_Aux_AADGroupNames = make_set(GroupName) by AzureResource_ResourceId, AzureResource_SubscriptionId, AzureResource_Aux_SubsName
+            | order by array_length(AzureResource_Aux_AADGroupIds) asc

```

Comment:
Generally we want to do Top 5 or 10 here, is the expectation that we won't have more than that normally?